Sources/SnapshotTesting/Common/View.swift

```diff
@@ -48,6 +48,9 @@ public struct ViewImageConfig {
   }
 
   #if os(iOS)
+  #if targetEnvironment(macCatalyst)
+  public static let mac = ViewImageConfig(size: .init(width: 480, height: 270))
```

Comment:
Can you please make this a static function instead so that the size can easily be customized?
```suggestion
  public static func macCatalyst(size: CGSize) -> ViewImageConfig {
    return ViewImageConfig(size: size)
  }
```